peripherals/flash/flash_latency16.yaml

```diff
@@ -0,0 +1,19 @@
+"FLASH":
+  ACR:
+    LATENCY:
+      WS0: [0, "0 wait states"]
+      WS1: [1, "1 wait states"]
+      WS2: [2, "2 wait states"]
+      WS3: [3, "3 wait states"]
+      WS4: [4, "4 wait states"]
+      WS5: [5, "5 wait states"]
+      WS6: [6, "6 wait states"]
+      WS7: [7, "7 wait states"]
+      WS8: [8, "8 wait states"]
+      WS9: [9, "9 wait states"]
+      WS10: [10, "10 wait states"]
+      WS11: [11, "11 wait states"]
+      WS12: [12, "12 wait states"]
+      WS13: [13, "13 wait states"]
+      WS14: [14, "14 wait states"]
+      WS15: [15, "15 wait states"]
```

Comment:
Do we even need this latency16 file? Looks like all users of flash_v1.yaml use the flash_latency8.yaml version.

Reply:
Looks like I planned to do something, but forgot.

Reply:
According to RM only 405/415 407/417 series in STM32F4 don't support higher states.

Reply:
Fixed